po/README.md

```diff
@@ -0,0 +1,19 @@
+# Translations generated by Launchpad
+
+## How to sync translations from Launchpad?
+- Run `update-pot` to generate latest translation template (`po/snappy.pot`).
+- Import generated translation template to snappy entry [here](https://translations.launchpad.net/snapd/trunk/+templates).
+- Wait until import is approved. You can monitor status in the import queue [here](https://translations.launchpad.net/snapd/trunk/+imports).
+- Export latest translations [here](https://translations.launchpad.net/snapd/trunk/+export) (download link will be sent to your email).
+- Extract archive: `tar -xvf launchpad-export.tar.gz`.
+- Create new directory to store processed translations `mkdir processed`.
+- `for file in snappy-*; do msgattrib --no-obsolete --translated --output-file="processed/${file#snappy-}" "$file"; done;`.
+    - Remove `snappy-` prefix.
+    - Remove obsolete #~ messages  `--no-obsolete`.
+    - Keep translated, remove untranslated messages `--translated`.
```

Comment:
Minor nitpick, reads slightly better like this to me:

```suggestion
    - Removes the `snappy-` prefix
    - `--no-obsolete` removes obsolete #~ messages
    - `--translated` excludes untranslated messages
```

Reply:
Agreed, updated